### ISSUE TITLE
Text is cut off by table of contents

### ISSUE THREAD

**Issue by CarlSinclair** [streetwriters/notesnook#6331]
### What happened?

Table of contents overlaps text, even though the left third of the screen is entirely empty. The table of contents should push the content to the left, not overlap it. Disabling margins takes advantage of this empty space, but doesn't account for the table of contents.

### Steps to reproduce the problem

Enter focus mode or fullscreen mode and enable table of contents

### Version

3.0.13-9019490-desktop

### Platform/OS

Windows

**Comment by Leviob** [comment]
Yes, I would love to see this improvement, as well as several other quality of life changes to the table of contents I outlined here: #4977
